src/Containers.jsx

```diff
@@ -433,7 +486,28 @@ class Containers extends React.Component {
                             <FormSelectOption value='all' label={_("All")} />
                         </FormSelect>
                     </ToolbarItem>
+                    <Divider isVertical />
```

Comment:
This will need a row-gap in custom CSS:
![Screen Shot 2021-10-21 at 09 44 17](https://user-images.githubusercontent.com/14921356/138233706-e71f88b2-1a63-4033-9522-265d1960879e.png)

or actually ideally an upstream issue.